src/modules/api/user/link/link.service.ts

```diff
@@ -6,4 +7,26 @@ import { Model } from 'mongoose';
 @Injectable()
 export class LinkService {
   constructor(@InjectModel(Link.name) private linkModel: Model<Link>) {}
+
+  async create(
+    name: any,
+    status: any,
+    customUrl: any,
+    redirectsTo: any,
+    inPool: any,
+    userId: any,
+  ): Promise<LinkDTO> {
```

Comment:
Why not mention all of the actual types?

Reply:
As we are using validator its ok even if we wont mention, but we should mention i feel

Reply:
But the dev should know what are the exact param types to be sent right...

Reply:
Class validator is just for validation and removing a middleware. So, **Whenever you declare a variable, it's type has to be mentioned**